packages/web/docs/src/content/gateway/deployment/docker.mdx

```diff
@@ -344,3 +344,45 @@ And finally start it (the config file is in the image and doesn't need to be mou
 ```sh
 docker run -p 4000:4000 hive-gateway-w-add-res supergraph
 ```
+
+## Known Security Vulnerabilities (affected CVE)
```

Comment:
The title says "(affected CVE)" but the actual list is of non-effective CVEs - maybe we can improve it?

Reply:
I agree, I struggled to find a title on this one, do you have ideas ?